### ISSUE TITLE
[Feature request] Could you create role grafana?

### ISSUE THREAD

**Issue by patsevanton** [buluma/ansible-role-grafana#1]
Hello! Thanks for role prometheus

Could you create role [grafana](https://grafana.com/)?

Populate role https://galaxy.ansible.com/cloudalchemy/grafana have many issue and many PR - last commit on Dec 10, 2020
Thanks!

**Comment by buluma** [comment]
there you go https://galaxy.ansible.com/buluma/grafana

**Comment by patsevanton** [comment]
issue blocked by https://github.com/buluma/ansible-role-prometheus/issues/5

**Comment by buluma** [comment]
this is already covered. try and pass variable to your playbook

```yaml
prometheus_components: [ "prometheus", "node_exporter", "alertmanager" ]
```

```yaml
---
- name: Install prometheus and Grafana
  hosts: grafana
  roles:
    - role: buluma.prometheus
      vars:
        prometheus_targets:
          node:
          - targets:
            - localhost:9100
        prometheus_components: [ "prometheus", "node_exporter", "alertmanager" ]
....
```

Output

```yaml
redirecting (type: connection) ansible.builtin.docker to community.docker.docker
changed: [grafana] => {"changed": true, "name": "grafana-server", "state": "started", "status": {"ActiveState": "active", "Description": "Grafana instance", "EnvironmentFile": "/etc/default/grafana-server", "Group": "grafana", "Id": "grafana-server.service", "JournalFile": "/var/log/journal/grafana-server.service.log", "JournalFilePath": "/var/log/journal/grafana-server.service.log", "KillMode": "control-group", "KillSignal": "SIGTERM", "LoadState": "loaded", "MainPID": "5734", "Names": "grafana-server.service", "NeedDaemonReload": "no", "NotifySocket": "/run/systemd/notify.grafana-server.service", "RemainAfterExit": "no", "RestartSec": "1s", "SendSIGHUP": "no", "SendSIGKILL": "yes", "StartLimitBurst": "5", "StartLimitIntervalSec": "10s", "StatusFile": "/run/grafana-server.service.status", "StatusFilePath": "/run/grafana-server.service.status", "SubState": "running", "TimeoutStartUSec": "1min 30s", "TimeoutStopUSec": "20s", "UnitFileState": "enabled", "User": "grafana", "WorkingDirectory": "/usr/share/grafana"}}
META: ran handlers
META: ran handlers

PLAY RECAP *******************************************************************************************************************************************************************
grafana                    : ok=50   changed=26   unreachable=0    failed=0    skipped=81   rescued=0    ignored=0 
```
